.github/workflows/main.yml

```diff
@@ -0,0 +1,108 @@
+name: build
+on:
+  push:
+    branches:
+      - main
+      - master
+      - poggenpower*
+  pull_request:
+    branches: 
+      - main
+      # comment
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: ${{ github.repository }}
+
+
+jobs:
+  checkout:
+    name: Checkout
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+#      - name: Setup pipeline enforcer
+#        uses: aquasecurity/pipeline-enforcer-action@v1.0.0
+#        with:
+#          aqua-key: ${{ secrets.AQUA_KEY }}
+#          aqua-secret: ${{ secrets.AQUA_SECRET }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@v2.1.0
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-PIPELINE-0008   
**Check Name:** Unrestricted dependency version   
**Severity:** MEDIUM   
**Message:** Dependency docker/login-action v2.1.0 version should be pinned to the commit sha  
[This comment was created by Aqua Pipeline]